### DISCUSSION TITLE
We have 4 stars

### DISCUSSION THREAD

**Discussion by Timothee-Cardoso** [scmn-dev/secman#12] [General]
# look @abdfnx 

![2021-03-28](https://user-images.githubusercontent.com/80710181/112753264-9417d180-8fdf-11eb-87f6-01ac0efd9058.png)

we've 4 stars 🤩

**Comment by abdfnx** [comment]
Wow, we did it, let's hit **1000** star 🌟

**Reply by Timothee-Cardoso** [reply]
let's do it bro 💪

**Reply by abdfnx** [reply]
Now we have 7 ⭐

**Reply by abdfnx** [reply]
and now `14` 🤯
